### PR TITLE
update data dump jobs (use read replica & set batch size)

### PR DESCRIPTION
**Pull request by larisa17**
Those changes were (partially) made manually in AWS . This fixes https://github.com/gitcoinco/passport/issues/2427

### CODE REVIEW

File: infra/aws/index.ts [hunk 1]
```diff
@@ -955,6 +955,8 @@ export const dailyDataDumpTaskDefinition = createScheduledTask(
       "python",
       "manage.py",
       "scorer_dump_data",
+      "--batch-size 1000",
```
Comment (nutrina) [inline infra/aws/index.ts:958]: "--batch-size" is duplicate ... it is set below as well

File: infra/aws/index.ts [hunk 2]
```diff
@@ -989,6 +991,8 @@ export const dailyDataDumpTaskDefinitionParquet = createScheduledTask(
       "python",
       "manage.py",
       "scorer_dump_data_parquet",
+      "--batch-size 1000",
```
Comment (nutrina) [inline infra/aws/index.ts:994]: duplicate batch-size ....

File: infra/aws/index.ts [hunk 3]
```diff
@@ -955,6 +955,8 @@ export const dailyDataDumpTaskDefinition = createScheduledTask(
       "python",
       "manage.py",
       "scorer_dump_data",
+      "--batch-size 1000",
+      "--database read_replica_0",
```
Comment (nutrina) [inline infra/aws/index.ts:959]: I am not 100% sure this will work ... can you specify with '=' in between?
  `--database=read_replica_0`,

**Review by nutrina — CHANGES_REQUESTED** [review]
Please see my comments.
There are duplicate specifications for batch-size, and some more minor things ...

File: infra/aws/index.ts (hunk 3 above)
Comment (larisa17) [reply infra/aws/index.ts]: FYI : Works both ways: `--database=read_replica_0` & `--database read_replica_0`(tested locally)

**Pull request merged**